### PR TITLE
fix: exchange rate process

### LINKED ISSUE #297: Invalid Rate Error
The worker service is not updating the exchange rate

HAPI log shows the following error:

image

### LINKED ISSUE #298: Change KYC Verification
Change the “verify KYC” action to work even if the account doesn’t display the real name. “Green Check” KYC

### PR DESCRIPTION
**Pull request by adriexnet**
### What does this PR do?

- Resolve #297
- Resolve #298

### Steps to test

1. first step
2. next step

#### CheckList

- [x] Follow proper Markdown format
- [x] The content is adequate
- [x] The content is available in both english and spanish
- [x] I Ran a spell check

### CODE REVIEW

File: contracts/affiliate/affiliate.abi [hunk 1]
```diff
@@ -380,87 +385,87 @@
         {
             "name": "addadmin",
             "type": "addadmin",
-            "ricardian_contract": "---\nspec_version: 0.1.0\ntitle: Add an Administrator User\nsummary: The intent of {{addadmin}} is to allow `admin` accounts to add a new admin user.\nicon: https://earnproton.com/icons/admin-user.png\n---"
+            "ricardian_contract": ""
```
Comment (xavier506) [inline contracts/affiliate/affiliate.abi:388]: you are removing the ricardian contracts

File: contracts/affiliate/affiliate.abi [hunk 2]
```diff
@@ -493,27 +498,8 @@
             "key_types": []
         }
     ],
-    "ricardian_clauses": [
```
Comment (xavier506) [inline contracts/affiliate/affiliate.abi:496]: lets restore the ricardian contract

File: contracts/affiliate/affiliate.abi (hunk 1 above)
Comment (adriexnet) [reply contracts/affiliate/affiliate.abi:388]: looks like on my local cant find the file

File: contracts/affiliate/affiliate.abi (hunk 2 above)
Comment (adriexnet) [reply contracts/affiliate/affiliate.abi:496]: @leisterfrancisco could you help us to regenerate the ricardian contract information?

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**